src/current/cockroachcloud/cloud-api.md

```diff
@@ -658,9 +784,13 @@ If the request was successful, the client will receive a response with the name
 
```

Comment:
I don't think we need this output.

Reply:
Streamlining and fixing braces instead of removing for now